cmd/download_test.go

```diff
@@ -114,7 +114,15 @@ func TestDownload(t *testing.T) {
 		err = runDownload(cfg, flags, []string{})
 		assert.NoError(t, err)
 
-		assertDownloadedCorrectFiles(t, filepath.Join(tmpDir, tc.expectedDir), tc.requestor)
+		targetDir := filepath.Join(tmpDir, tc.expectedDir)
+		assertDownloadedCorrectFiles(t, targetDir, tc.requestor)
+
+		metadata := `{"track":"bogus-track","exercise":"bogus-exercise","id":"bogus-id","url":"","handle":"alice","is_requester":%s,"auto_approve":false}`
+		metadata = fmt.Sprintf(metadata, tc.requestor)
+		path := filepath.Join(targetDir, "bogus-track", "bogus-exercise", ".solution.json")
+		b, err := ioutil.ReadFile(path)
+		assert.NoError(t, err)
+		assert.Equal(t, metadata, string(b), "the solution metadata file")
```

Comment:
Actually we should be cleaning up files here `os.RemoveAll(filepath.join(targetDir, "bogus-track")`

Reply:
Good catch, thank you!